bigframes/session/__init__.py

```diff
@@ -527,7 +527,6 @@ def _read_gbq_colab(
         query = bigframes.core.pyformat.pyformat(
             query,
             pyformat_args=pyformat_args,
-            # TODO: add dry_run parameter to avoid API calls for data in pyformat_args
```

Comment:
Note: I probably will do this for the pandas use case, but didn't need it for bigframes.